buildconfig/pygame-stubs/_sdl2/touch.pyi

```diff
@@ -0,0 +1,7 @@
+from typing import Dict, Union
+
+
+def get_num_devices() -> int: ...
+def get_device(index: int) -> int: ...
+def get_num_fingers(touch_id: int) -> int: ...
```

Comment:
`touch.c@pg_touch_num_fingers` does not use a `touch_id` keyword for get_num_fingers.

Reply:
If this is device_id that I fixed it